javaslang/src/main/java/javaslang/collection/BitMappedTrie.java

```diff
@@ -148,29 +189,37 @@ private static int treeSize(int branchCount, int depthShift) {
 
     T get(int index) {
         final T[] leaf = getLeaf(index);
-        final int leafIndex = lastDigit(offset() + index);
+        final int leafIndex = lastDigit(offset + index);
         return leaf[leafIndex];
     }
 
     /**
      * fetch the leaf, corresponding to the given index.
      * Node: the offset and length should be taken into consideration as there may be leading and trailing garbage.
+     * Also, the returned array is mutable, but should not be mutated!
      */
     @SuppressWarnings("unchecked")
     T[] getLeaf(int index) {
```

Comment:
this is also `private` now, but since it's accessed from `iterator`, this way we can avoid the bridge